boards/frdm-kw41z/include/gpio_params.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2017 SKF AB
```

Comment:
Did you write this file for a different company?

Reply:
Yes, it was part of a project for a client